src/Components/Shared/src/ComponentParametersTypeCache.cs

```diff
@@ -26,7 +26,7 @@ internal class ComponentParametersTypeCache
             }
         }
 
-        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026", Justification = "This pattern is not linker friendly.")]
+        [RequiresUnreferencedCode("This type attempts to load component parameters that may be linked away.")]
```

Comment:
The official term is "trim", we try to avoid using "link" in any public facing names/strings since that has typically a different meaning for developers.